apps/demos/Demos/Chat/Customization/jQuery/data.js

```diff
@@ -0,0 +1,53 @@
+const getTimestamp = function(date, offsetMinutes = 0) {
+    return date.getTime() + offsetMinutes * 60000;
+}
+
+const date = new Date();
+date.setHours(0, 0, 0, 0);
+
+const currentUser = {
+    id: "c94c0e76-fb49-4b9b-8f07-9f93ed93b4f3",
+    name: "John Doe",
+};
+
+const supportAgent = {
+    id: "d16d1a4c-5c67-4e20-b70e-2991c22747c3",
+    name: "Support Agent",
+    avatarUrl: "https://raw.githubusercontent.com/DevExpress/DevExtreme/refs/heads/24_2/apps/demos/images/petersmith.png",
+};
+
+let messages = [
+    {
+        timestamp: getTimestamp(date, -9),
+        author: supportAgent,
+        text: "Hello, John!\nHow can I assist you today?"
+    },
+    {
+        timestamp: getTimestamp(date, -7),
+        author: currentUser,
+        text: "Hi, I'm having trouble accessing my account."
+    },
+    {
+        timestamp: getTimestamp(date, -7),
+        author: currentUser,
+        text: "It says my password is incorrect."
+    },
+    {
+        timestamp: getTimestamp(date, -7),
+        author: supportAgent,
+        text: "I can help with that. Can you please confirm your UserID for security purposes?"
```

Comment:
[according to the tech writer's last notes]
```suggestion
        text: "I can help you with that. Can you please confirm your UserID for security purposes?"
```

Reply:
[a788963](https://github.com/DevExpress/DevExtreme/pull/28441/commits/a7889633ac3296ccaa25a0589dccff72c053b179)